src/useReplicant/index.ts

```diff
@@ -1,24 +1,37 @@
 import {useEffect, useState} from 'react';
 import clone from 'lodash.clone';
 
+export interface UseReplicantOptions {
+	namespace: string;
+}
+
 /**
  * Subscribe to a replicant, returns tuple of the replicant value and `setValue` function.
  * The component using this function gets re-rendered when the value is updated.
  * The `setValue` function can be used to update replicant value.
  * @param repName The name of the replicant to use
  * @param initialValue Initial value to pass to `useState` function
+ * @param options Options object.  Currently supports the optional `namespace` option
  */
 export const useReplicant = <T>(
 	repName: string,
 	initialValue: T,
+	options?: UseReplicantOptions,
 ): [T, (newValue: T) => void] => {
 	// Local state to store replicant value
 	const [value, updateValue] = useState<T>(initialValue);
 
 	// Declare replicant
-	const replicant = nodecg.Replicant(repName, {
-		defaultValue: initialValue,
-	});
+	let replicant: any;
```

Comment:
If I remember correctly, the function `nodecg.Replicant()` returns a value with type `Replicant` (confusing but that's what happens...). It may be possible to avoid `any` here.

Reply:
Ok. I'll check that out.  I wasn't aware of any of the nodecg types.